doc/tutorial/single_cell_allen.rst

```diff
@@ -0,0 +1,273 @@
+.. _tutorialsinglecellallen:
+
+A single cell model from the Allen Brain Atlas
+==============================================
+
+In this tutorial we'll see how we can take a model from the `Allen Brain Atlas <https://brain-map.org/>`_
+and run it with Arbor.
+
+.. Note::
+
+   **Concepts covered in this example:**
+
+   1. Take a model from an online atlas.
+   1. Load a morphology from an ``swc`` file.
+   2. Load a parameter fit file and apply it to a :class:`arbor.decor`.
+   4. Building a :class:`arbor.cable_cell` object.
+   5. Building a :class:`arbor.single_cell_model` object.
+   6. Running a simulation and visualising the results.
+
+Obtaining the model
+-------------------
+
+We need a model for which morphological data is available. We'll take
+`a cell from the mouse's visual cortex <http://celltypes.brain-map.org/experiment/electrophysiology/488683425>`_.
+As the ``README`` in the "Biophysical - all active" model (zip) file explains, the data set is created for use with the Allen SDK and the
+Neuron simulator. Instructions about running the model with the
+Allen SDK and Neuron `can be found here <https://allensdk.readthedocs.io/en/latest/biophysical_models.html>`_.
+For your convenience, this tutorial comes with a reference trace pre-generated.
+
+In the "Biophysical - all active" model (zip) file you'll find:
+
+* a directory with mechanisms (modfiles). For your convenience, Arbor already comes with the
+  :func:`arbor.allen_catalogue`, containing mechanisms used by the Allen Brain Atlas.
+* an ``swc`` file describing the morphology of the cell.
+* ``fit_parameters.json`` describing a set of optimized mechanism parameters. This information will need to be
+  loaded into the Arbor simulation.
+
+We will replicate the "Sweep 35" experiment, which applies a current of 150 nA for a duration of 1 s.
+
+The morphology
+--------------
+
+:ref:`In an earlier tutorial <tutorialsinglecellswc-cell>` we've seen how an ``swc`` file can be loaded **(1)**
+and how the labels can be set **(2)**:
+
+.. code-block:: python
+
+  # (1) Load the cell morphology.
+  morphology = arbor.load_swc_allen('single_cell_allen.swc', no_gaps=False)
```

Comment:
Outdated.

Reply:
This function does not exist anymore.

Reply:
All references corrected in subsequent commits.